app/src/main/res/layout/context_menu.xml

```diff
@@ -22,6 +26,10 @@
         android:background="#ff262626"
         android:textColor="#ff8b8b8b"
```

Comment:
Can you make this white with 40% opacity? via https://github.com/mozilla-mobile/focus-android/issues/1260#issuecomment-336601194

Reply:
Okay